common/djangoapps/enrollment/management/commands/enroll_user_in_course.py

```diff
@@ -0,0 +1,56 @@
+"""
+Management command for enrolling a user into a course via the enrollment api
+"""
+from optparse import make_option
```

Comment:
Nit: optparse is deprecated.  We can now use `argparse` with django 1.8.  You can see how it's used  in this pending PR: https://github.com/edx/edx-platform/pull/11198/files#diff-4c497748ec77404ef8d240c08a447490